.github/workflows/build-pr.yml

```
@@ -0,0 +1,98 @@
+name: build-pr
+
+on:
+  push:
+    branches:
+      - develop
+      - master
+  pull_request:
+
+jobs:
+  build-zip:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check out source files
+        uses: actions/checkout@v2
+
+      - name: Get Composer cache directory
+        id: composer-cache
+        run: |
+          echo "::set-output name=dir::$(composer config cache-files-dir)"
+
+      - name: Configure Composer cache
+        uses: actions/cache@v1
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-
+
+      - name: Install Composer dependencies
+        # Scripts are not ignored as they are needed to apply patches for the
+        # `sabberworm/php-css-parser` dependency.
+        run: composer install --prefer-dist --optimize-autoloader
+
+      - name: Setup Node.js (.nvmrc)
+        uses: actions/setup-node@v1
+        with:
+          # nvm aliases are not yet supported, see https://github.com/actions/setup-node/issues/26.
+          node-version: 12
+
+      - name: Configure npm cache
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-node-cache-
+
+      - name: Install Node dependencies
+        # Prevent malicious scripts from being run with `--ignore-scripts`
+        run: npm install --ignore-scripts
+
+      - name: Create destination directory
+        run: mkdir -p ${{ github.ref }}
+
+      - name: Build develop version
+        run: |
+          npm run build:dev
+          mv amp.zip ${{ github.ref }}/amp-dev.zip
+
+      - name: Build production version
+        run: |
+          npm run build:prod
+          mv amp.zip ${{ github.ref }}/amp.zip
+
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v1
```

There's a newer version available here:
```suggestion
        uses: actions/upload-artifact@v2
```